geometry_msgs/Pose[] waypoints
---
trajectory_msgs/JointTrajectory trajectory